src/components/List/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -18390,6 +18390,7 @@ exports[`List should render correctly with autoClose and ExpendableContent 1`] =
       <details
         class="cache-163f8eq-StyledRow e1r829126"
         data-testid="row-0"
+        open=""
```

Comment:
Is that normal?

Reply:
Yes, the previous test was not working :)